modules/core/src/controllers/orbit-controller.js

```diff
@@ -293,4 +312,10 @@ export default class OrbitController extends Controller {
   constructor(props) {
     super(OrbitState, props);
   }
+
+  _getTransitionProps(event) {
+    // Enables Transitions on double-tap.
+    const props = event.type === 'doubletap' ? LINEAR_TRANSITION_PROPS : NO_TRANSITION_PROPS;
```

Comment:
Is this necessary? The transition would not be triggered unless zoom is changed.